From b4c740f6a059c40912fbc226c3ae0ff59d5c5d58 Mon Sep 17 00:00:00 2001
From: Hector Dearman <hjd@google.com>
Date: Wed, 21 Mar 2018 11:55:19 +0000
Subject: [PATCH 164/306] Rename persist.traced.enable to
 sys.traced.enable_override

Allowing for both a default in the system
image and an override form the server requires at least two properties.
Given that we're using two properties anyway we can change the
server override to match the pattern of the other flags and not be
a persisted setting.

Test: Manual.
Bug: b/76077784
Merged-In: I98932085430cdeaf4ccd036a0751f119c0cdeb3c
Change-Id: I98932085430cdeaf4ccd036a0751f119c0cdeb3c
(cherry picked from commit c5f06ac35e9a1d7d209820c0543c24892e990949)
---
 .../com/android/server/am/GlobalSettingsToPropertiesMapper.java | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/services/core/java/com/android/server/am/GlobalSettingsToPropertiesMapper.java b/services/core/java/com/android/server/am/GlobalSettingsToPropertiesMapper.java
index 690d985ef09..5bf5020c99e 100644
--- a/services/core/java/com/android/server/am/GlobalSettingsToPropertiesMapper.java
+++ b/services/core/java/com/android/server/am/GlobalSettingsToPropertiesMapper.java
@@ -43,7 +43,7 @@ class GlobalSettingsToPropertiesMapper {
         {Settings.Global.FPS_DEVISOR, ThreadedRenderer.DEBUG_FPS_DIVISOR},
         {Settings.Global.DISPLAY_PANEL_LPM, "sys.display_panel_lpm"},
         {Settings.Global.SYS_UIDCPUPOWER, "sys.uidcpupower"},
-        {Settings.Global.SYS_TRACED, "persist.traced.enable"},
+        {Settings.Global.SYS_TRACED, "sys.traced.enable_override"},
     };
 
 
-- 
2.17.1

